### PR TITLE
Implement exec bee

### PR DESCRIPTION
**Pull request by dschmidt**

### CODE REVIEW

File: bees/execbee/execbee.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *    Authors:
+ *      Dominik Schmidt <domme@tomahawk-player.org>
+ */
+
+// beehive's IRC module.
```
Comment (muesli) [inline bees/execbee/execbee.go]: Really? ;-)

File: bees/execbee/execbee.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+					errScanner := bufio.NewScanner(errReader)
+					go func() {
+						for errScanner.Scan() {
+							foo := errScanner.Text()
+							log.Println("execbee: err: | ", foo)
+							errBuffer = append(errBuffer, foo)
+						}
+					}()
+
+					cmd.Start()
```
Comment (muesli) [inline bees/execbee/execbee.go]: Start may fail - e.g. if a command doesn't exist. Should check for error returned for Start() and Wait() below, too.

File: bees/execbee/execbee.go [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+					cmd.Wait()
+
+					ev := bees.Event{
+						Bee:  mod.Name(),
+						Name: "cmd",
```
Comment (muesli) [inline bees/execbee/execbee.go]: Should maybe be named differently. "commandResult" or something?

Comment (dschmidt) [inline bees/execbee/execbee.go]: I called it that way because the struct is called ```cmd``` in go too

Comment (dschmidt) [inline bees/execbee/execbee.go]: https://golang.org/pkg/os/exec/#Cmd

**Pull request merged**